dcat/index.html

```diff
@@ -4635,6 +4635,13 @@ <h3>Compressed and packaged distributions</h3>
 <section id="changes" class="appendix">
     <h2>Change history</h2>
     <p>A full change-log is available on <a href="https://github.com/w3c/dxwg/commits/gh-pages/dcat">GitHub</a></p>
+    <section id="changes-since-20140116">
+        <h3>Changes since the W3C Recommendation of 4 February 2020</h3>
+        <p>The document has undergone the following changes since the W3C Recommendation of 4 February 2020 [[?VOCAB-DCAT-2-20200204]]:</p>
+        <li>
+        Section <a href="#dataset-versions"></a> was extended with draft guidelines to deal with version delta (<a href="https://github.com/w3c/dxwg/issues/#89">Issue #89</a>), version release date (<a href="https://github.com/w3c/dxwg/issues/#91">Issue #91</a>),  version identifier (<a href="https://github.com/w3c/dxwg/issues/#92">Issue #92</a>),  version compatibility (<a href="https://github.com/w3c/dxwg/issues/#1258">Issue #1258</a>) and resource status (<a href="https://github.com/w3c/dxwg/issues/#1238">Issue #1238</a>). 
+        </li>
+    </section>
     <section id="changes-since-20140116">
         <h3>Changes since the W3C Recommendation of 16 January 2014</h3>
         <p>The document has undergone the following changes since the W3C Recommendation of 16 January 2014 [[?VOCAB-DCAT-20140116]]:</p>
```

Comment:
```suggestion
        <p>The document has undergone the following changes since the  DCAT 2014 W3C Recommendation of 16 January 2014 [[?VOCAB-DCAT-20140116]]:</p>
```